src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -270,8 +270,8 @@ export class NetCoreDebugHelper implements DebugHelper {
             const inspectInfo = (await ext.runWithDefaults(client =>
                 client.inspectContainers({ containers: [containerName] })
             ))?.[0];
-            const containerInfo = inspectInfo ? JSON.parse(inspectInfo.raw) : undefined;

```

Comment:
Has something changed that we no longer need to parse `inspectInfo`?

Reply:
In the core `DockerClient` parsing was already happening, but not all fields were being paid attention to. The raw JSON output was also attached to the inspection result for further consumption by callers. With isolation mode being added to the first layer of parsing, the second is no longer needed.